docs/reference/config.md

```diff
@@ -1375,6 +1375,9 @@ The following settings are available:
 `singularity.envWhitelist`
 : Comma separated list of environment variable names to be included in the container environment.
 
+`singularity.libraryDir`
+: Directory where remote Singularity images are retrieved. When using a computing cluster it must be a shared folder accessible to all compute nodes.
+
 `singularity.noHttps`
```

Comment:
it should be added an entry for `apptainer.libraryDir` as well. 

Also the description could be extended to make it coherent with the explanation [here](https://www.nextflow.io/docs/latest/container.html)

> When looking for a Singularity image file, Nextflow first checks the library directory, and if the image file is not found, the cache directory is used as usual. The library directory can be defined either using the NXF_SINGULARITY_LIBRARYDIR environment variable or the singularity.libraryDir configuration setting (the latter overrides the former).